### PR TITLE
[main] Use quay images instead of ko

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: openshift/patches/use_quay_images.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
++        image: quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+ ---
+ apiVersion: sources.knative.dev/v1
+ kind: SinkBinding
+@@ -114,7 +114,7 @@ spec:
+     spec:
+       containers:
+       - name: single-heartbeat
+-        image: ko://knative.dev/eventing/test/test_images/heartbeats
++        image: quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.13.2
```
Comment (matzew) [inline openshift/patches/use_quay_images.patch:19]: sure we wanna use that images? (version)

Comment (pierDipi) [reply openshift/patches/use_quay_images.patch:19]: Yes, we tested this script with this version.

I don't want to verify it with newer versions for now.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**